### DISCUSSION TITLE
[PlayerError] client is missing "GUILD_VOICE_STATES" intent

### DISCUSSION THREAD

**Discussion by iiAlanii** [Androz2091/discord-player#741] [Q&A] [answered]
When I updated to the newest release of discord-player version 5.1.0, I get "[PlayerError] client is missing "GUILD_VOICE_STATES" intent".

**Comment by twlite** [comment, marked as answer]
@iiAlanii hello, you need to enable `GUILD_VOICE_STATES` on your discord client:

```js
const client = new Discord.Client({
    intents: ["GUILD_VOICE_STATES", ...someOtherIntents]
});
```

**Reply by iiAlanii** [reply]
Hello @DevAndromeda !
Thank you for the reply. Yeah, sorry for not reading the update log on Discord-player before I got here. I read that the discord-player has completely changed.

But again, tank you for your reply!
